bodhi/templates/fragments.html

```diff
@@ -214,6 +214,9 @@ <h4 class="modal-title" id="markdown-help-label">Fedora-Flavored Markdown</h4>
               <p>You can reference <strong>bugzilla tickets</strong> by simply prefixing a ticket id with the <code>&#35;</code> character.</p>
               <pre><code>We really need #1185409</code></pre>
               <p>... we will automatically generate a link to the ticket in place.</p>
+              <p>You can also specify the <strong>ticket tracker</strong> if your update is fixing a ticket reported elsewhere than in the Fedora bugzilla.</p>
```

Comment:
We should re-work the docs just above this that say you can use the old un-prefixed format.